core/domain/fs_domain.py

```diff
@@ -471,6 +550,64 @@ def isfile(self, filepath):
         except cloudstorage.NotFoundError:
             return False
 
+    def get_file_content(self, filepath):  # pylint: disable=unused-argument
```

Comment:
Why not just use the existing get()?

Reply:
Because we don't have get implemented for GCS. We directly fetch the file by the Public url for that entity https://storage.googleapis.com/bucket_name ....